src/web/nginx.conf

```diff
@@ -54,31 +54,38 @@ http {
     ssl_protocols       TLSv1.2 TLSv1.3;
     ssl_prefer_server_ciphers   on;
     ssl_ciphers "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH+aRSA HIGH !RC4 !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS";
-    # disabling unwanted http methods
+    
+    # Disabling unwanted http methods
     if ($request_method !~ ^(DELETE|GET|POST)$ ){
       return 405;
     }
-    # prevent clickjack attacks
-    add_header X-Frame-Options "SAMEORIGIN";
-    # prevent XSS attacks
-    add_header X-XSS-Protection "1; mode=block";
-    # Configuring HSTS
-    add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload';
+
+    # If user agent is a program/bot, block
+    if ($http_user_agent ~* "java|curl|python") {
```

Comment:
do we need/want this? If we're trying to debug something, I can see us using curl, maybe python. probably also not sustainable to block list of languages